bitcoin/bitcoind/wallet.go

```diff
@@ -77,13 +79,15 @@ func NewBitcoindWallet(mnemonic string, params *chaincfg.Params, repoPath string
 		masterPrivateKey: mPrivKey,
 		masterPublicKey:  mPubKey,
 		binary:           binary,
+		controlPort:      torControlPort,
+		userTor:          useTor,
```

Comment:
Good catch. That's a typo that it looks like the ide replaced everywhere. I'll fix.